libs/island-ui/core/src/lib/Select/Select.css.ts

```diff
@@ -287,6 +287,7 @@ export const menu = style(
         borderBottomLeftRadius: 8,
         borderBottomRightRadius: 8,
         boxSizing: 'border-box',
+        zIndex: 9999,
```

Comment:
I don't think we have something like that at a "global" level. As you can see from the screenshot, the Toast and Navigation ui components just have large z indexes since they are used globally
![image](https://user-images.githubusercontent.com/43557895/191941303-b7dc5ec1-ec60-47d6-aa4b-55f323c04390.png)

Reply:
Haha. At least there's room for improvement, which would be quite easy to do imo. I can also see that the service portal already did some work like this for their code.